### PR TITLE
Hello Microverse Project: Milestone 1: Create main content

### PR DESCRIPTION
**Pull request by andres-condezo**
## Hello Microverse Project: Milestone 1

In this milestone we:
- Initialized the repository.
- Added HTML file.
- Added CSS file.
- Added linter file.
- Added "gitignore" file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,23 @@
...
+
+👤 **Andres Condezo**
+
+- GitHub: [@andres-condezo](https://github.com/andres-condezo)
+- Twitter: [@andres_condezo](https://twitter.com/andres_condezo)
+- LinkedIn: [andres-condezo](https://linkedin.com/in/andres-condezo)
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (gabrielyea) [inline README.md:23]: - [x] It seems you forgot to add the MIT file! Please fix the broken link :link: . You can find the file in this [template](https://github.com/microverseinc/readme-template#readme)
Remember that having an MIT license is part of microverse standards and helps your project look more professional. :star:

**Review by gabrielyea — CHANGES_REQUESTED** [review]
# Changes required! :recycle: 
Hello @andres-condezo, 
You are almost there!

Great job! :star: 
First let me point out: 
- Good commit history! :heavy_check_mark: 
- You are using gitflow! :heavy_check_mark: 
- Linter pass! :heavy_check_mark: 

You just need to fix some minor details! Easy work! :muscle: 

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (andres-condezo) [reply README.md:23]: Right!, now I upload it.

**Review by gabrielyea — APPROVED** [review]
# Approved! :partying_face: :tada: 
Hello again @andres-condezo, 


You resolved all required changes! :star: :star: :star: 
What are you waiting for?! Merge it already!


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**